ql/math/interpolations/multicubicspline.hpp

```diff
@@ -493,7 +493,7 @@ namespace QuantLib {
       private:
         const SplineGrid &grid_;
         const data_table &y_;
-        const std::vector<bool> &ae_;
+        const std::vector<bool> ae_;
```

Comment:
If the default initializer is the only issue, it's likely more efficient to create a private static variable and use it for the default instead of copying the vector.